addons/calendar/wizard/calendar_provider_config.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.addons.base.models.ir_module import assert_log_admin_access
+
+
+class CalendarProviderConfig(models.TransientModel):
+    _name = 'calendar.calendar.provider.config'
+    _description = 'Calendar Provider Configuration Wizard'
+
+    external_calendar_provider = fields.Selection([
+        ('google', 'Google'), ('microsoft', 'Outlook')],
+        "Choose an external calendar to configure", default='google')
+
+    # Allow to sync with eventually existing ICP keys without creating them if respective module is not installed
+    # Using same field names and strings as their respective res.config.settings
+    cal_client_id = fields.Char(
```

Comment:
Why not ``google_calendar_client_id`` to keep a direct naming match with the config parameter ? (same for other field) ?

Reply:
As written above: `Using same field names and strings as their respective res.config.settings` ?

Reply:
Ok